!!python/object:huggingface_hub.community.DiscussionWithDetails
author: genzml
conflicting_files: null
created_at: 2023-06-22 23:07:37+00:00
diff: null
endpoint: https://huggingface.co
events:
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: /avatars/68d1699d6d24855a49a862960ff870d7.svg
      fullname: Kunal Shanishchara
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: genzml
      type: user
    createdAt: '2023-06-23T00:07:37.000Z'
    data:
      edited: false
      editors:
      - genzml
      hidden: false
      identifiedLanguage:
        language: en
        probability: 0.7877318263053894
      isReport: false
      latest:
        author:
          avatarUrl: /avatars/68d1699d6d24855a49a862960ff870d7.svg
          fullname: Kunal Shanishchara
          isHf: false
          isPro: false
          name: genzml
          type: user
        html: '<p>I am using ml.g5.12xlarge instance to deploy. The model deployments
          fails during health checks with the below error</p>

          <p>sagemaker.exceptions.UnexpectedStatusException: Error hosting endpoint
          huggingface-pytorch-tgi-inference-2023-06-22-22-26-05-836: Failed. Reason:
          The primary container for production variant AllTraffic did not pass the
          ping health check. Please check CloudWatch logs for this endpoint..</p>

          '
        raw: "I am using ml.g5.12xlarge instance to deploy. The model deployments\
          \ fails during health checks with the below error\r\n\r\nsagemaker.exceptions.UnexpectedStatusException:\
          \ Error hosting endpoint huggingface-pytorch-tgi-inference-2023-06-22-22-26-05-836:\
          \ Failed. Reason: The primary container for production variant AllTraffic\
          \ did not pass the ping health check. Please check CloudWatch logs for this\
          \ endpoint.."
        updatedAt: '2023-06-23T00:07:37.128Z'
      numEdits: 0
      reactions: []
    id: 6494e249d6ac9642831e71c6
    type: comment
  author: genzml
  content: "I am using ml.g5.12xlarge instance to deploy. The model deployments fails\
    \ during health checks with the below error\r\n\r\nsagemaker.exceptions.UnexpectedStatusException:\
    \ Error hosting endpoint huggingface-pytorch-tgi-inference-2023-06-22-22-26-05-836:\
    \ Failed. Reason: The primary container for production variant AllTraffic did\
    \ not pass the ping health check. Please check CloudWatch logs for this endpoint.."
  created_at: 2023-06-22 23:07:37+00:00
  edited: false
  hidden: false
  id: 6494e249d6ac9642831e71c6
  type: comment
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: https://aeiljuispo.cloudimg.io/v7/https://cdn-uploads.huggingface.co/production/uploads/636d18755aaed143cd6698ef/AalDh13Gp8jv1BfM5IASh.png?w=200&h=200&f=face
      fullname: Philipp Singer
      isHf: false
      isOrgMember: true
      isOwner: false
      isPro: false
      name: psinger
      type: user
    createdAt: '2023-06-23T08:25:38.000Z'
    data:
      edited: false
      editors:
      - psinger
      hidden: false
      identifiedLanguage:
        language: en
        probability: 0.9744317531585693
      isReport: false
      latest:
        author:
          avatarUrl: https://aeiljuispo.cloudimg.io/v7/https://cdn-uploads.huggingface.co/production/uploads/636d18755aaed143cd6698ef/AalDh13Gp8jv1BfM5IASh.png?w=200&h=200&f=face
          fullname: Philipp Singer
          isHf: false
          isPro: false
          name: psinger
          type: user
        html: "<p>Hi <span data-props=\"{&quot;user&quot;:&quot;genzml&quot;}\" data-target=\"\
          UserMention\" class=\"SVELTE_PARTIAL_HYDRATER contents\">\n\n<span class=\"\
          inline-block\"><span class=\"contents\"><a href=\"/genzml\">@<span class=\"\
          underline\">genzml</span></a></span>\n\n\t</span></span> - unfortunately\
          \ I have no experience with sagemaker. Might be good to check with them\
          \ to see if they know what's up.</p>\n"
        raw: Hi @genzml - unfortunately I have no experience with sagemaker. Might
          be good to check with them to see if they know what's up.
        updatedAt: '2023-06-23T08:25:38.068Z'
      numEdits: 0
      reactions: []
    id: 64955702d622eec1c695b599
    type: comment
  author: psinger
  content: Hi @genzml - unfortunately I have no experience with sagemaker. Might be
    good to check with them to see if they know what's up.
  created_at: 2023-06-23 07:25:38+00:00
  edited: false
  hidden: false
  id: 64955702d622eec1c695b599
  type: comment
- !!python/object:huggingface_hub.community.DiscussionStatusChange
  _event:
    author:
      avatarUrl: https://aeiljuispo.cloudimg.io/v7/https://cdn-uploads.huggingface.co/production/uploads/636d18755aaed143cd6698ef/AalDh13Gp8jv1BfM5IASh.png?w=200&h=200&f=face
      fullname: Philipp Singer
      isHf: false
      isOrgMember: true
      isOwner: false
      isPro: false
      name: psinger
      type: user
    createdAt: '2023-07-03T19:05:17.000Z'
    data:
      status: closed
    id: 64a31bedfbd6fdd06d72cf24
    type: status-change
  author: psinger
  created_at: 2023-07-03 18:05:17+00:00
  id: 64a31bedfbd6fdd06d72cf24
  new_status: closed
  type: status-change
is_pull_request: false
merge_commit_oid: null
num: 1
repo_id: h2oai/h2ogpt-gm-oasst1-en-2048-falcon-40b-v1
repo_type: model
status: closed
target_branch: null
title: Anyone facing issues deploying this model on Sagemaker
